src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -279,11 +280,50 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
       }
     }
 
+  def withRawlsUser[T](userRef: RawlsUserRef, txn: RawlsTransaction)(op: (RawlsUser) => T): T = {
+    containerDAO.authDAO.loadUser(userRef, txn) match {
+      case Some(user) => op(user)
+      case None => throw new RawlsException(s"Couldn't find user for userRef $userRef")
+    }
+  }
+
+  def withRawlsGroup[T](groupRef: RawlsGroupRef, txn: RawlsTransaction)(op: (RawlsGroup) => T): T = {
+    containerDAO.authDAO.loadGroup(groupRef, txn) match {
+      case Some(group) => op(group)
+      case None => throw new RawlsException(s"Couldn't find group for groupRef $groupRef")
+    }
+  }
+
```

Comment:
have realised these should probably give RequestCompletes rather than throw exceptions